openwisp_monitoring/db/backends/__init__.py

```diff
@@ -57,7 +64,18 @@ def load_backend_module(backend_name=TIMESERIES_DB['BACKEND'], module=None):
                 "Try using 'openwisp_monitoring.db.backends.XXX', where XXX is one of:\n"
                 f"{builtin_backends}"
             ) from e
+        else:
+            raise e
 
 
-timeseries_db = load_backend_module(module='client').DatabaseClient()
+if '2' in TIMESERIES_DB['BACKEND']:
+    timeseries_db = load_backend_module(module='client').DatabaseClient(
+        bucket=TIMESERIES_DB['BUCKET'],
+        org=TIMESERIES_DB['ORG'],
+        token=TIMESERIES_DB['TOKEN'],
+        url=f"http://{TIMESERIES_DB['HOST']}:{TIMESERIES_DB['PORT']}",
+    )
+else:
+    timeseries_db = load_backend_module(module='client').DatabaseClient()
```

Comment:
Any backend specific configuration should be done in the backend class. Perform this operation in the `__init__` method of the backend class 

```
.DatabaseClient(
        bucket=TIMESERIES_DB['BUCKET'],
        org=TIMESERIES_DB['ORG'],
        token=TIMESERIES_DB['TOKEN'],
        url=f"http://{TIMESERIES_DB['HOST']}:{TIMESERIES_DB['PORT']}",
 ```